client/test/cypress/support/fragments/demoproject.js

```diff
@@ -0,0 +1,12 @@
+export const reset = () => {
+  cy.get('body').then(body => {
+    if (body.find('[data-nrt="deleteDemo"]').length >  0) {
+      cy.GetByDataNrt('deleteDemo').click()
+      // TODO(Issue224) : Replace the basic Modal component of iviewui by a custom one
```

Comment:
Why not remove it completely ? Isn't that what issues are for ? If you're afraid to forget where to update this code, you can specify it directly in the issue you just created ;)

Reply:
Two purpose : 

* Add a strong link between the code location and the issue in GitHub (you know, in case you don't have access to GitHub (ie. offline mode), you still have the Todo to tells you that something needs to be done)
* If someone decide to fix it just by reading the code, he/she knows that this todo also have an issue related to it, so he/she can resolve it directly :)